test/functional/wallet_abandonconflict.py

```diff
@@ -40,21 +48,21 @@ def run_test(self):
 
         sync_blocks(self.nodes)
         newbalance = self.nodes[0].getbalance()
-        assert(balance - newbalance < Decimal("0.001")) #no more than fees lost
+        assert balance - newbalance < Decimal("0.001")  #no more than fees lost
         balance = newbalance
 
         # Disconnect nodes so node0's transactions don't get into node1's mempool
         disconnect_nodes(self.nodes[0], 1)
 
         # Identify the 10btc outputs
-        nA = next(i for i, vout in enumerate(self.nodes[0].getrawtransaction(txA, 1)["vout"]) if vout["value"] == Decimal("10"))
-        nB = next(i for i, vout in enumerate(self.nodes[0].getrawtransaction(txB, 1)["vout"]) if vout["value"] == Decimal("10"))
-        nC = next(i for i, vout in enumerate(self.nodes[0].getrawtransaction(txC, 1)["vout"]) if vout["value"] == Decimal("10"))
+        nA = next(d["vout"] for d in self.nodes[0].gettransaction(txA)["details"] if d["amount"] == Decimal("10"))
```

Comment:
`d` is short for `details`

Reply:
What is a `detail` in Bitcoin?

Reply:
The name `details` can't be changed in the RPC since it's part of the API. I'm just suggesting you could use a more variable name which is more descriptive for people reading the test.

ACK either way.

Reply:
+1 descriptiveness within test :)